### ISSUE TITLE
Insufficient color contrast ratio on events page

### ISSUE THREAD

**Issue by gramliu** [CMU-313/fall-2021-hw2-connect5#16]
The current lighthouse score for accessibility on the /events/events/ page is 89. This can be improved by improving the contrast ratio between the background and foreground colors for certain elements.

See [CMU-313/Mayan-EDMS#15](https://github.com/CMU-313/Mayan-EDMS/issues/15)
